generator/lib/src/code_builder.dart

```diff
@@ -29,6 +30,13 @@ class CodeBuilder extends Builder {
   @override
   FutureOr<void> build(BuildStep buildStep) async {
     // build() will be called only twice, once for the `lib` directory and once for the `test` directory
+    Pubspec? pubspec;
+    try {
+      final pubspecFile = File(path.join(dir(buildStep), '../pubspec.yaml'));
```

Comment:
Not that familiar with pubspec, but can the flutter libs dependency also come from another file (external variable or import)? Or put differently: can this fail to find flutter libs, even though it's a dependency?

Reply:
I don't know of any other way (which doesn't mean there isn't any, though I doubt it). But it's fine even if it fails - you would just get the plain dart's `openStore()` version and would have to try and figure out why it doesn't open the directory, potentially creating a GH issue here.